### PR TITLE
Add information about local blueprint.

### PR DESCRIPTION
**Pull request by mshima**

### CODE REVIEW

File: modules/creating_a_blueprint.md [hunk 1]
```diff
@@ -68,6 +68,16 @@ export default class extends ClientGenerator {
 }
 ```
 
+## Local Blueprints
+
+A local blueprint is implemented inside your project `.blueprint` directory. It's detected and used by default.
```
Comment (mraible) [inline modules/creating_a_blueprint.md:73]:
```suggestion
A local blueprint is implemented inside your project's `.blueprint` directory. It's detected and used by default.
```

**Comment by mraible** [comment]
Change URL to customizing instead of customizaing

**Comment by mshima** [comment]
Do you mean https://github.com/jhipster/jhipster.github.io/pull/1224?

**Comment by mraible** [comment]
Yep!

**Review by mraible — APPROVED** [review]

**Comment by deepu105** [comment]
Awesome

**Pull request merged**